kensu/pandas/data_frame.py

```diff
@@ -486,8 +486,20 @@ def k_to_format(self, result_regex, *args, **kwargs):
         #FIXME
         table = None
         if fmt == 'gbq' or isinstance(args[0],str) == False:
-          return None
-
+            return None
+        if fmt == 'sql':
+            from sqlalchemy.engine.base import Engine
+            engine = None
+            for arg in args:
+                if isinstance(arg,Engine):
+                    engine = arg
+            if engine == None:
+                engine = kwargs['con']
+            if engine.name == 'postgresql':
+                fmt = 'Postgres table'
+
+                #FIXME How to find the schema name?
+                location = "postgres://"+ engine.url.host +":"+ str(engine.url.port)+'/'+engine.url.database + '.public.' + args[0]
```

Comment:
here we'll need smarter logic (e.g. could sometimes `args[0]` already include `db schema name`), but good for now.